spec/controllers/catalog_controller_spec.rb

```diff
@@ -54,6 +54,51 @@
       end
     end
   end
+  describe 'POST submit_cart_updates_form on item' do
+    before(:each) do
+      @equipment_model = FactoryGirl.create(:equipment_model)
+      put :add_to_cart, id: @equipment_model.id
+    end
+    it 'should adjust item quantity' do
+      # check if cart contains an item
+      expect session[:cart].items
```

Comment:
I think you're missing the actual expectation here (e.g. `expect(session[:cart].items).not_to be_empty` or something like that)